src/schema/me/followed_shows.ts

```diff
@@ -0,0 +1,54 @@
+import { ShowType } from "schema/show"
+import { IDFields } from "schema/object_identification"
+
+import { pageable, getPagingParameters } from "relay-cursor-paging"
+import { connectionDefinitions, connectionFromArraySlice } from "graphql-relay"
+import { GraphQLObjectType } from "graphql"
+
+const FollowedShowEdge = new GraphQLObjectType({
+  name: "FollowedShowEdge",
+  fields: {
+    partner_show: {
+      type: ShowType,
+    },
+    ...IDFields,
+  },
+})
+
+export const FollowedShowConnection = connectionDefinitions({
+  name: "FollowedShowConnection",
+  // FIXME: 'edgeType' does not exist in type 'ConnectionConfig'
+  // @ts-ignore
```

Comment:
I'm not clear on this — here and throughout, what's the significance of this comment? Is `connectionDefinitions` doing anything with this extra arg you are providing?